user_docs/en/changes.md

```diff
@@ -64,7 +64,7 @@ To use this feature, "allow NVDA to control the volume of other applications" mu
 * NVDA will no longer announce Windows 11 clipboard history entries when closing the window while items are present. (#17308, @josephsl)
 * If the plugins are reloaded while a browseable message is opened, NVDA will no longer fail to report subsequent focus moves. (#17323, @CyrilleB79)
 * When using applications such as Skype, Discord, Signal and Phone Link for audio communication, NVDA speech and sounds no longer decrease in volume. (#17349, @jcsteh)
-* Opening the Python console will no longer fail in case an error occurs while retrieving snapshot variables.
+* Opening the NVDA Python Console will no longer fail in case an error occurs while retrieving snapshot variables.
```

Comment:
```suggestion
* Opening the NVDA Python Console will no longer fail in case an error occurs while retrieving snapshot variables. (#17391, @CyrilleB79)
```